lib/usd/utils/tests/test_MayaTransformAPI.cpp

```diff
@@ -0,0 +1,1196 @@
+#include <gtest/gtest.h>
+#include <fstream>
+#include "../MayaTransformAPI.h"
+#include <pxr/usd/usdGeom/xform.h>
+
+const char* const fullXformDef = 
+R"(#
+usda 1.0
+(
+    defaultPrim = "maya_xform"
+)
+
+def Xform "maya_xform"
+{
+    float3 xformOp:rotateXYZ = (4, 5, 6)
+    float3 xformOp:rotateXYZ:rotateAxis = (10, 11, 12)
+    float3 xformOp:scale = (7, 8, 9)
+    matrix4d xformOp:transform:shear = ( (1, 0, 0, 0), (0.1, 1, 0, 0), (0.2, 0.3, 1, 0), (0, 0, 0, 1) )
+    float3 xformOp:translate:rotatePivot = (13, 14, 15)
+    float3 xformOp:translate:rotatePivotINV = (-13, -14, -15)
+    float3 xformOp:translate:rotatePivotTranslate = (0.19292736, 0.6936933, -0.8563779)
+    float3 xformOp:translate:scalePivot = (16, 17, 18)
+    float3 xformOp:translate:scalePivotINV = (-16, -17, -18)
+    float3 xformOp:translate:scalePivotTranslate = (142, 167.6, 144)
+    double3 xformOp:translate = (1, 2, 3)
+    uniform token[] xformOpOrder = [
+        "xformOp:translate", 
+        "xformOp:translate:rotatePivotTranslate", 
+        "xformOp:translate:rotatePivot",
+        "xformOp:rotateXYZ",
+        "xformOp:rotateXYZ:rotateAxis",
+        "xformOp:translate:rotatePivotINV",
+        "xformOp:translate:scalePivotTranslate",
+        "xformOp:translate:scalePivot",
+        "xformOp:transform:shear",
+        "xformOp:scale", 
+        "xformOp:translate:scalePivotINV"]
+}
+)";
+
+TEST(MayaTransformAPI, orderCheckIsValid)
+{
+  #if AL_SUPPORT_LEGACY_NAMES
```

Comment:
A little confused as to why this is testing the AL_SUPPORT_LEGACY_NAMES.  The tests look very similar to the ones starting at line 153.